coderd/coderd.go

```diff
@@ -350,7 +351,8 @@ func New(options *Options) *API {
 			r.Get("/state", api.workspaceBuildState)
 		})
 	})
-	r.NotFound(api.siteHandler.ServeHTTP)
+
+	r.NotFound(gziphandler.GzipHandler(http.HandlerFunc(api.siteHandler.ServeHTTP)).ServeHTTP)
```

Comment:
Any reason to prefer `gziphandler` vs `chi/middleware.Compress` here? The Chi one also supports `deflate`. (I think it would be nice to support all of: deflate, gzip and br (brotli), but chi does not handle br.)